dotnet/src/SemanticKernel.Abstractions/AI/TextToImage/ImageContent.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.SemanticKernel.TextToImage;
+
+/// <summary>
+/// Represents image content.
+/// </summary>
+public class ImageContent : ContentBase
```

Comment:
This should be sealed. If a developer was a customized `ImageContent` they should extend `ContentBase` directly.